src/core/utils.js

```diff
@@ -166,6 +166,6 @@ exports.fromTimestamp = fromTimestamp;
 exports.getMantissaDecimalString = getMantissaDecimalString;
 exports.getMantissa16FromString = getMantissa16FromString;
 
-exports.sjcl = require('sjcl-extended');
+exports.sjcl = require('sjcl-codecs');
```

Comment:
Yes, definitely temporary, just a sketch. What do you think about an npm
package with a lite sjcl build with a unique name though? I didn't want to
just depend on sjcl and a postinstall script because it can be misleading
to require "sjcl" then.

Having said that, could just add a './sjclcodecs.js' module that adds some
indirection.

I wish sjcl were more like lodash and had highly modular npm packages.

Hrmmm?